src/index.js

```diff
@@ -435,6 +435,9 @@ io.on('connection', function(socket) {
       case 'attack': res = room.game.attack(action.cardId, params[0]); break;
       case 'invokeMonsterEffect': res = room.game.invokeMonsterEffect(action.cardId, params); break;
 
+      case 'summonTribute1': res = room.game.summon(action.cardId, params[1], params[2], params[3], [params[0]]); break;
+      case 'summonTribute2': res = room.game.summon(action.cardId, params[2], params[3], params[4], [params[0], params[1]]); break;
```

Comment:
These two lines are too long

Reply:
We do not limit the character count for one line since we believe that long lines are easier to read under some circumstances.

Please see our Eslint config:
https://github.com/DotLab/touhou-card-game-nodejs/blob/98569c8bc43dc3c211a42dee737da77b954faffe/.eslintrc.json#L14

The refactor did make the line shorter and easier to read nevertheless.